axelrod/tests/strategies/test_cycler.py

```diff
@@ -45,13 +46,31 @@ class TestCycler(TestPlayer):
 
         def test_strategy(self):
             """Starts by cooperating"""
-            for i in range(20):
-                responses = itertools.islice(itertools.cycle(cycle), i)
-            self.responses_test(responses)
+            match_len = 20
+            actions_cycle = _get_actions_cycle_against_cooperator(cycle)
+            test_actions = list(itertools.islice(itertools.cycle(actions_cycle), match_len))
+            self.versus_test(axelrod.Cooperator(), test_actions)
 
     return TestCycler
```

Comment:
do you mean the blank before return TestCycler (line 53)? I didn't add it, but i do like it.  I think it nicely separates the instructions to construct the class from the act of returning that class.  it made it easy for me to read.  

if you meant the two blank lines after the factoryclass.  my IDE tries to enforce two blank lines between a class and whatever comes after.  I assume it's convention